src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -289,14 +308,39 @@ class ThetaStarPathPlanner : public PathPlanner
                       const Coordinate &end);
 
     /**
-     * Checks for line of sight between Coordinates
+     * Checks for line of sight between Coordinates using Bresenham's Line Algorithm
+     * https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm
      *
-     * @param coord1 The first Coordinate
-     * @param coord2 The second Coordinate
+     * @param coord0 The first Coordinate
+     * @param coord1 The second Coordinate
+     *
+     * @return true if line of sight from coord0 to coord1
+     */
+    bool lineOfSight(const Coordinate &coord0, const Coordinate &coord1);
+
+    /**
+     * Supplementary method for lineOfSight to check for line of sight when the slope
+     * between the two coordinates is between 0 and 1
+     * https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm#All_cases
+     *
+     * @param coord0 The first Coordinate
+     * @param coord1 The second Coordinate
      *
-     * @return true if line of sight from coord1 to coord2
+     * @return true if line of sight from coord0 to coord1
      */
-    bool lineOfSight(const Coordinate &coord1, const Coordinate &coord2);
+    bool checkLineLow(const Coordinate &coord0, const Coordinate &coord1);
```

Comment:
Thoughts on pulling checkLineLow/High into a geom/algorithms file. I suppose that'd mean that Coordinate would also need to live in geom